nitime/analysis/tests/test_granger.py

```diff
@@ -109,5 +109,10 @@ def test_GrangerAnalyzer():
     # Test inputting ij:
     g2 = gc.GrangerAnalyzer(ts1, ij=[(0, 1), (1, 0)])
 
+    # g1 agrees with g2
+    npt.assert_almost_equal(g1.causality_xy[0, 1], g2.causality_xy[0, 1])
```

Comment:
But they're arranged so that one analyzer's `x` is the other analyzer's `y`, which is why the test below requires that `xy[0, 1]` for one analyzer be the same as `yx[1, 0]` for the other analyzer.

I am not saying that were correctly arranged before this change, but we need tests that demonstrate more convincingly what the error is, and how it is fixed through the proposed change.

Thanks for taking a look, by the way!